src/module-rebuilder.ts

```diff
@@ -120,25 +120,26 @@ export class ModuleRebuilder {
     const buildLocation = path.resolve(this.modulePath, 'build', this.rebuilder.buildType);
 
     d('searching for .node file', buildLocation);
-    const buildLocationFiles = await fs.readdir(buildLocation);
+    const buildLocationFiles = await fs.promises.readdir(buildLocation);
     d('testing files', buildLocationFiles);
 
     const nodeFile = buildLocationFiles.find((file) => file !== '.node' && file.endsWith('.node'));
     const nodePath = nodeFile ? path.resolve(buildLocation, nodeFile) : undefined;
 
-    if (nodePath && await fs.pathExists(nodePath)) {
+    if (nodePath && fs.existsSync(nodePath)) {
       d('found .node file', nodePath);
       if (!this.rebuilder.disablePreGypCopy) {
         const abiPath = path.resolve(this.modulePath, `bin/${this.rebuilder.platform}-${this.rebuilder.arch}-${this.rebuilder.ABI}`);
         d('copying to prebuilt place:', abiPath);
-        await fs.mkdir(abiPath, { recursive: true });
-        await fs.copyFile(nodePath, path.join(abiPath, `${this.nodeGyp.moduleName}.node`));
+        await fs.promises.mkdir(abiPath, { recursive: true });
+        await fs.promises.copyFile(nodePath, path.join(abiPath, `${this.nodeGyp.moduleName}.node`));
       }
     }
   }
 
   async writeMetadata(): Promise<void> {
-    await fs.outputFile(this.metaPath, this.metaData);
+    await fs.promises.mkdir(path.dirname(this.metaPath), { recursive: true });
+    await fs.promises.writeFile(this.metaPath, this.metaData);
```

Comment:
Since this is a `graceful-fs` call, it'll need to be passed through `util.promisify`